tests/HashTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace LanSuite\Tests;
+
+use LanSuite\PasswordHash;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+use function PHPUnit\Framework\assertEquals;
+
+class HashTest extends TestCase
+{
+
+    protected function setUp() : void
+    {
+        $GLOBALS['cfg'] = [];
+          error_reporting(E_ALL);
```

Comment:
```suggestion
```
PHPUnit has its own error handling and we should use this: https://docs.phpunit.de/en/11.2/error-handling.html